distributed/distributed-schema.yaml

```diff
@@ -803,44 +803,47 @@ properties:
   rmm:
     type: object
     description: |
-      Configuration options for the RAPIDS Memory Manager
+      Configuration options for the RAPIDS Memory Manager.
     properties:
       pool-size:
-        type:
-        - integer
-        - "null"
-        description:
-          The size of the memory pool in bytes
+        type: [integer, 'null']
+        description: |
+          The size of the memory pool in bytes.
   ucx:
     type: object
     description: |
-      UCX provides access to other network interconnects like Infiniband and NVLINK
+      UCX provides access to other transport methods including NVLink and InfiniBand.
     properties:
+      cuda_copy:
+        type: boolean
+        description: |
+          Set environment variables to enable CUDA transfers over UCX, even if InfiniBand and
+          NVLink are not supported or disabled.
       tcp:
-        type:
-        - boolean
-        - "null"
+        type: boolean
+        description: |
+          Set environment variables to enable TCP over UCX, even if InfiniBand and NVLink
+          are not supported or disabled.
       nvlink:
-        type:
-        - boolean
-        - "null"
+        type: boolean
+        description: |
+          Set environment variables to enable UCX over NVLink, requires use of UCX protocol
```

Comment:
These are all UCX configurations, so "requires use of UCX protocol" is redundant, plus not all configurations add that sentence. I suggest removing that.